src/js/messaging/reducers/messages.js

```diff
@@ -7,21 +7,23 @@ import {
   CLEAR_DRAFT,
   DELETE_DRAFT_ATTACHMENT,
   FETCH_THREAD_SUCCESS,
-  SEND_MESSAGE_SUCCESS,
   TOGGLE_MESSAGE_COLLAPSED,
   TOGGLE_MESSAGES_COLLAPSED,
   TOGGLE_MOVE_TO,
   TOGGLE_REPLY_DETAILS,
   UPDATE_DRAFT
 } from '../utils/constants';
 
+import { composeMessage } from '../config';
+
 const initialState = {
   data: {
     message: null,
     thread: [],
     draft: {
       attachments: [],
-      body: makeField('')
+      body: makeField(''),
+      charsRemaining: composeMessage.maxChars.message
```

Comment:
The import of `composeMessage` and all instances of `charsRemaining` came back from the merge.